pkg/operation/hybridbotanist/controlplane.go

```diff
@@ -180,32 +157,6 @@ func (b *HybridBotanist) DeployETCD() error {
 		}
 
 		if err := b.ApplyChartSeed(filepath.Join(chartPathControlPlane, "etcd"), b.Shoot.SeedNamespace, fmt.Sprintf("etcd-%s", role), nil, etcd); err != nil {
-			if role == common.EtcdRoleMain {
```

Comment:
/cc @swapnilgm - I think it's fine to remove this code already and release it with 0.26.0, right? If so, you don't need to change anything here, @stoyanr.